modules/serverless-event-source-metrics.adoc

```diff
@@ -0,0 +1,24 @@
+[id="serverless-event-source-metrics_{context}"]
+= Event source metrics
+
+You can use the following metrics to verify that events have been delivered from the event source to the connected event sink.
```

Comment:
```suggestion
You can use the following metrics to verify the delivery of events from the event source to the connected event sink.
```

Simplification purposes. Ignore if it changes the meaning of the sentence.

Reply:
I honestly prefer it as is, I think this suggestion would make it a little harder to understand